SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -310,7 +311,31 @@ void USpatialNetDriver::CreateServerWorkerEntity()
 	Components.Add(EntityAcl(WorkerIdPermission, ComponentWriteAcl).CreateEntityAclData());
 	Components.Add(interest.CreateInterestData());
 
-	Connection->SendCreateEntityRequest(MoveTemp(Components), nullptr);
+	Worker_RequestId RequestId = Connection->SendCreateEntityRequest(MoveTemp(Components), nullptr);
+	CreateEntityDelegate OnCreateWorkerEntityResponse;
+	OnCreateWorkerEntityResponse.BindLambda([this](const Worker_CreateEntityResponseOp& Op)
+	{
+		if (Op.status_code != WORKER_STATUS_CODE_SUCCESS)
+		{
+			if (++CreateServerWorkerEntityNumberOfAttempts <= SpatialConstants::MAX_NUMBER_COMMAND_ATTEMPTS)
+			{
+				UE_LOG(LogSpatialOSNetDriver, Warning, TEXT("Worker entity creation request failed and will retry: \"%s\""),
+					UTF8_TO_TCHAR(Op.message));
+
+				FTimerHandle RetryTimer;
+				TimerManager.SetTimer(RetryTimer, [this]()
+				{
+					CreateServerWorkerEntity();
+				}, SpatialConstants::GetCommandRetryWaitTimeSeconds(CreateServerWorkerEntityNumberOfAttempts), false);
+			}
+			else
+			{
+				UE_LOG(LogSpatialOSNetDriver, Error, TEXT("Worker entity creation request failed too many times. (%u attempts)"),
+					SpatialConstants::MAX_NUMBER_COMMAND_ATTEMPTS)
+			}
```

Comment:
personal pref which just leads to less indenting / conditional braces, if the second clause comes first with an early exit  `return;` statement it's a but more readable:

```
if (CreateServerWorkerEntityNumberOfAttempts == SpatialConstants::MAX_NUMBER_COMMAND_ATTEMPTS)
{
      UE_LOG(LogSpatialOSNetDriver, Error, TEXT("Worker entity creation request failed too many times. (%u attempts)"), SpatialConstants::MAX_NUMBER_COMMAND_ATTEMPTS);
      return;
}

CreateServerWorkerEntityNumberOfAttempts ++
UE_LOG(LogSpatialOSNetDriver, Warning, TEXT("Worker entity creation request failed and will retry: \"%s\""),
					UTF8_TO_TCHAR(Op.message));
// etc...
```

Reply:
Also this shouldn't compile regardless as 334 is missing a semi-colon